runtime/lua/vim/lsp/handlers.lua

```diff
@@ -364,7 +364,9 @@ function M.hover(_, result, ctx, config)
   local markdown_lines = util.convert_input_to_markdown_lines(result.contents)
   markdown_lines = util.trim_empty_lines(markdown_lines)
   if vim.tbl_isempty(markdown_lines) then
-    vim.notify('No information available')
+    if config.silent ~= true then
```

Comment:
can there just  `if config.silent then`  ? although it doesn't matter...

Reply:
Isn't that inconsistent with the check above?